docs/ibc-relayer.md

```diff
@@ -0,0 +1,109 @@
+# IBC relaying guide
+
+Babylon uses [IBC](https://ibcprotocol.dev/)
+(Inter-Blockchain Communication protocol) to enable cross-chain
+communication with other Cosmos SDK chains. This guide focuses on the specific
+configurations needed
+when relaying with Babylon, particularly around its unique unbonding period
+mechanism.
+
+## Important Note on Babylon's Unbonding Period
+
+Babylon has a unique unbonding mechanism that differs from standard Cosmos SDK
+chains. The Babylon Genesis chain disables the standard `x/staking` module and
+wraps it with the `x/epoching` module, introducing secure, fast unbonding
+through Bitcoin timestamping.
+
+> **Important**: The standard `x/staking` module's unbonding time parameter
+> remains at the default 21 days, but **this value should be ignored** when
+> configuring the relayer's trusting period.
+
+1. **Epoching System**:
+    - All staking operations and voting power adjustments are processed at the
+      final block of each epoch
+    - The final block of each epoch is checkpointed onto the Bitcoin blockchain
```

Comment:
```suggestion
    - The metadata of each epoch is checkpointed onto the Bitcoin blockchain
```

Technically the AppHash of the last block of the epoch is checkpointed, and this Apphash is derived from the entire execution trace prior to that block